src/components/dashboard/Rewards.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { RaisedButton, FloatingActionButton, FontIcon, FlatButton } from 'material-ui'
+
+import SplitSection from './SplitSection'
+import ProgressSection from './ProgressSection'
+
+import './Rewards.scss'
+
+class Rewards extends React.Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return (
+      <div styleName="root">
+        <SplitSection title="Rewards"
+          head={(
+            <div styleName="title">
+              <h3>Rewards</h3>
+            </div>
+          )}
+          foot={(
```

Comment:
We also have Vote section, it will also use the split section.
I prefer to have ability to customize header, so we can pass a structured header.